packages/runner-ct/src/app/app.scss

```diff
@@ -135,59 +135,36 @@ main.app-ct {
 }
 
 
-// styles for react-split-pane
+// styles for react-split-pane and custom ResizableBox
 .Resizer {
```

Comment:
Should we have this in a separate css file? 
It seems it is associated with a separate component.